Metadata for all other Overture Maps tables

xavipereztr opened the pull request.

xavipereztr committed:
Update catalog.jsonnet

Updated catalog.jsonnet with all the rest of OM tables.

xavipereztr committed:
Upload the jsonnet files of all reminding tables

xavipereztr committed:
Uploaded examples for all the reminding OM tables

xavipereztr committed:
Fixed keyword

simonff commented on examples/overture-maps/overture-maps_addresses_address.js:
```diff
@@ -0,0 +1,10 @@
+var dataset = ee.FeatureCollection(
```
please use loadBigQueryTable()

simonff replied:
I mean, just like you use loadBigQueryTable() in https://github.com/google/earthengine-catalog/blob/main/examples/overture-maps/overture-maps_places_place.js#L2

If that doesn't work for some of the datasets, let's discuss why

xavipereztr committed:
Merge branch 'google:main' into xavipereztr-alltables

xavipereztr committed:
Fixed regular examples to include loadBigQueryTable() and previews

- Regular examples: ee.FeatureCollection() → ee.FeatureCollection.loadBigQueryTable()
- Preview files: Changed to SQL query pattern with ee.FeatureCollection.runBigQuery()

xavipereztr committed:
Merge branch 'google:main' into xavipereztr-alltables

xavipereztr commented on examples/overture-maps/overture-maps_addresses_address.js:
```diff
@@ -0,0 +1,10 @@
+var dataset = ee.FeatureCollection.loadBigQueryTable(
```
@simonff All our tables are already clustered on **geometry**. 

<img width="1188" height="1884" alt="Image" src="https://github.com/user-attachments/assets/5b05a674-448d-4155-a11e-0cac6d17c104" />

xavipereztr committed:
Overture Maps: fix preview thumbnails by enlarging example viewports.

The previous 0.1°x0.1° NYC box used by every example rendered an
empty/solid-gray thumbnail for most datasets because too few features
overlapped it. Widen to an NYC-metro bbox (~0.6°x0.6° around the 5
boroughs + NJ) for all datasets except places_place and base_bathymetry.

base_bathymetry is moved to Florida + Bahamas shallow waters and gets
a shallow→deep blue palette, since NYC is inland and bathymetry is
ocean-only.

xavipereztr commented on examples/overture-maps/overture-maps_base_bathymetry.js:
```diff
@@ -0,0 +1,10 @@
+var dataset = ee.FeatureCollection.loadBigQueryTable(
```
Please check new changes.

xavipereztr reviewed (commented):
@simonff Could you please validate the thumbnails again? thanks!
